### PR TITLE
[TE] rootcause - prevent debounce error message

### PR DESCRIPTION
**Pull request by apucher**

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js [hunk 1]
```diff
@@ -786,6 +788,8 @@ export default Controller.extend({
     const { anomalyUrn, anomalyFeedback, sessionText } =
       this.getProperties('anomalyUrn', 'anomalyFeedback', 'sessionText');
 
+    if (!anomalyUrn) { return; }
+
     // debounce: do not run if destroyed
     if (this.isDestroyed) { return; }
```
Comment (aaronucsd) [inline thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: this vs line 764 `if (this.isDestroyed)`?

Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: hmm?

Comment (aaronucsd) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: maybe dup. We can remove one?

Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: why would this be a dup? sometime rca loads without an anomaly id, bt controller is not destroyed either

Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: resolved offline

**Pull request merged**